1.process_data/nbconverted/process_jump_pilot.py

```diff
@@ -5,22 +5,22 @@
 
 # ## Imports
 
-# In[1]:
+# In[ ]:
```

Comment:
Just putting a note here even though you probably already know, but make sure the notebook associated to the file is ran prior to nbconvert.

Reply:
Will do! See initial comment